client-guide/heartbeat-meeting.html.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Heartbeat meeting
+meta_description: >
+  TODO
+---
+
+We aim to have this meeting quarterly. You will talk with our Team lead about our relationship.
+
+## Goals
+
+The main goals of this recurring event are:
+
+- understand if our relationship is strong;
+- understand what's not going well in the collaboration;
+- understand if you need any adjustment to the team (additional team members, particular skill sets, etc.) in the next period;
+
+## Template
+
+You can use the following questions to understand the meeting content and to prepare for the discussion:
+
+- Are you generally satisfied with our collaboration?
+- Which kind of growth do you expect in the next quarter? What about one year?
```

Comment:
I would also use "What" instead of "Which" here, since "Which" is for choosing between a predefined number of options.